### ISSUE TITLE
Scroll back to top button needs a valid label

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2453]
OCIO occurrence 14%, severity 10

Violation: "Provide a valid label for form fields"

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/58733dce-c61a-4f2a-8952-7a35f12b42f2)
